packages/solid-sample/src/consts/index.ts

```diff
@@ -0,0 +1,7 @@
+export const baseUrl = window.location.origin;
+export const redirectUrl = `${baseUrl}/callback`;
+
+export const appId = 'hi9w1eijf2nlnuz88xjt6'; // Register the sample app in Logto dashboard
```

Comment:
we can replace them with env variables.

Reply:
Or just consts, like in the other samples. Not a big deal, though.